TIL/topics/dart_basics/VariousDataFormats.md

```
@@ -0,0 +1,120 @@
+		```dart
+		class Person {
+			final String name;
+			final int age;
+
+			Person({required this.name, required this.age});
+			...
+			// 직렬화 Dart의 맵과 동일한 형태이기 때문에 map으로 만듬
+			Map<String, dynamic> toJson() 
+				=> {'name': this.name, 'age': this.age};
+		}
+		void main() {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)